frontend/src/components/Footer.tsx

```diff
@@ -75,23 +74,26 @@ const Footer = () => {
     <USWDSFooter
       data-testid="footer"
       size="medium"
-      primary={<div />}
```

Comment:
If we pass this `null` instead of an empty div, it doesn't put an empty div in the container.